charts/memgraph/templates/statefulset.yaml

```diff
@@ -27,57 +27,120 @@ spec:
         {{- toYaml . | nindent 4 }}
       {{- end }}
     spec:
+      terminationGracePeriodSeconds: 1800
       securityContext:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      volumes:
+        - name: memgraph-lib-storage
+          persistentVolumeClaim:
+          {{- if .Values.persistentVolumeClaim.existingClaim }}
+            claimName: {{ .Values.persistentVolumeClaim.existingClaim }}
+          {{- else}}
+            claimName: {{ include "memgraph.fullname" . }}-lib-storage
+          {{- end }}
+      {{- if .Values.persistentVolumeClaim.createLogStorage }}
+        - name: memgraph-log-storage
+          persistentVolumeClaim:
+            claimName: {{ include "memgraph.fullname" . }}-log-storage
+      {{- end }}
       containers:
         - name: memgraph
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           args:
           {{- range .Values.memgraphConfig }}
           - {{ . | quote }}
           {{- end }}
+          {{- if not .Values.persistentVolumeClaim.createLogStorage }}
+          - "--log-file=''"
+          {{- end }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           securityContext:
             runAsUser: 0
           ports:
-            - name: memgraph
+            - name: bolt
               containerPort: {{ .Values.service.port }}
+            - name: websocket
+              containerPort: 7444
+            - name: http
+              containerPort: 9091
+          startupProbe:
+            tcpSocket:
+              port: bolt
+            failureThreshold: 240
+            periodSeconds: 30
+          livenessProbe:
+            tcpSocket:
+              port: bolt
+            periodSeconds: 30
+            timeoutSeconds: 20
+          readinessProbe:
+            tcpSocket:
+              port: bolt
+            periodSeconds: 30
           {{- with .Values.resources }}
           resources:
             {{- toYaml . | nindent 12 }}
           {{- end }}
+          env:
+            - name: MEMGRAPH_USER
+              value: {{ .Values.memgraphUser | default "memgraph"}}
+            - name: MEMGRAPH_PASSWORD
+              value: {{ .Values.memgraphPassword | default "memgraph" }}
+            {{ if .Values.memgraphEnterpriseLicense }}
+            - name: MEMGRAPH_ENTERPRISE_LICENSE
+              value: {{ .Values.memgraphEnterpriseLicense }}
+            {{- end}}
+            - name: MEMGRAPH_ORGANIZATION_NAME
+              value: {{ .Values.memgraphOrganizationName | default "" }}
```

Comment:
Let's leave it like this, for now; I will probably introduce Secrets in a few days.